WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingViewModel.swift

```diff
@@ -27,23 +32,48 @@ final class BlazeAdDestinationSettingViewModel: ObservableObject {
         return String(format: Localization.finalDestination, buildFinalDestinationURL())
     }
 
+    // View model for the add parameter view.
+    var blazeAddParameterViewModel: BlazeAddParameterViewModel {
+        let urlParameter = selectedParameterIndex != nil ? parameters[selectedParameterIndex!] : nil
+
+        return BlazeAddParameterViewModel(
+            remainingCharacters: calculateRemainingCharacters(),
+            isNotFirstParameter: parameters.count > 1,
```

Comment:
Instead of adding `isNotFirstParameter`, how about subtracting 1 to the remaining character count due to the required "&"?

Reply:
I don't fully understand this but it got me thinking and I did a similar refactor on

https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/4b5a52ffda1871e892c7ab2352e191af80ea229f

let me know what you think.

Reply:
Yup this was what I meant.